lib/chartmogul/csv/uploads/cancellation.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+module ChartMogul
+  module CSV
+    module Uploads
+      class Cancellation < Base
+        set_resource_path 'data_platform/v1/data_sources/:data_source_uuid/uploads'
```

Comment:
This can be in base for now at least, no?

Reply:
no.

Reply:
Why? The `Base` class is in `CSV:Uploads`.

Reply:
```
medali@medali ~ $ irb
2.7.0 :001 > 
2.7.0 :002 > class A
2.7.0 :003 >   class << self
2.7.0 :004 >     def set_a(value)
2.7.0 :005 >       @a = value
2.7.0 :006 >     end
2.7.0 :007 >     
2.7.0 :008 >     def get_a
2.7.0 :009 >       @a
2.7.0 :010 >     end
2.7.0 :011 >   end
2.7.0 :012 > end
 => :get_a 
2.7.0 :013 > 
2.7.0 :014 > class B < A
2.7.0 :015 >   set_a("Hello")
2.7.0 :016 > end
 => "Hello" 
2.7.0 :017 > 
2.7.0 :018 > class C < B
2.7.0 :019 > end
 => nil 
2.7.0 :020 > B.get_a
 => "Hello" 
2.7.0 :021 > C.get_a
 => nil 
2.7.0 :022 > 

```

Reply:
OK, I see your point now :heavy_check_mark: